proxy/logging/LogConfig.cc

```diff
@@ -616,15 +616,9 @@ LogConfig::update_space_used()
         //
         // then check if the candidate belongs to any given log type
         //
-        ts::TextView type_name(entry->d_name, strlen(entry->d_name));
-        // A rolled log will look something like:
-        // squid.log_some.hostname.com.20191029.18h15m02s-20191029.18h30m02s.old
-        //
-        // The following logic cuts things back to original unrolled file which
-        // is the key into the deleting_info map (squid.log in the above example).
-        auto suffix = type_name;
-        type_name.remove_suffix(suffix.remove_prefix(suffix.find('.') + 1).remove_prefix(suffix.find('_')).size());
-        auto iter = deleting_info.find(type_name);
+        ts::TextView rolled_filename(entry->d_name, strlen(entry->d_name));
```

Comment:
Do we need to use Ts:TextView, since we are not parsing the string here anymore, when a std::string_view would do?

Reply:
If your'e going to be that dedicated to extirpating all traces of `TextView`, you might as well go all the way and not have a variable at all, since it's only used once.

Reply:
Both good points.